v2.1/create-security-certificates-custom-ca.md

```diff
@@ -0,0 +1,117 @@
+`node.crt`   | Server certificate
+`node.key`   | Key for server certificate
+`ui.crt` | UI certificate
+`ui.key` | Key for the UI certificate
+
+### Client key and certificates
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate
+`client.<username>.crt` | Client certificate for `<username>` (e.g., `client.root.crt` for user `root`)
+`client.<username>.key` | Key for the client certificate
+
+## Split node certificates
+
+The node certificate discussed in the `cockroach cert` command documentation is multifunctional, which means the same certificate is presented when the node acts as a server as well as a client. To make the certificate multi-functional, the `node.crt` created using the `cockroach cert` command has `CN=node` and the list of IP addresses and DNS names listed in `Subject Alternative Names` field. This works if you are also using the CockroachDB CA created using the `cockroach cert` command. However, if you need to use an external public CA or your own organizational CA, the CA policy might not allow it to sign a node certificate containing a CN that is not an IP address or domain name.
+
+To get around this issue, we can split the node key and certificate into two:
+
+- `node.crt` and `node.key`: The node certificate to be presented when the node acts as a server and the corresponding key. `node.crt` must have the list of IP addresses and DNS names listed in `Subject Alternative Names`.
+- `client.node.crt` and `client.node.key`: The node certificate to be presented when the node acts as a client for another node, and the corresponding key. `client.node.crt` must have `CN=node`.
+
+### Node key and certificates
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate
+`node.crt`   | Node certificate for when node acts as server. <br><br>Must have the list of IP addresses and DNS names listed in `Subject Alternative Names`
+`node.key`   | Key corresponding to `node.crt`
+`client.node.crt` | Node certificate for when node acts as client. <br><br>Must have `CN=node`.
+`client.node.key` | Key corresponding to `client.node.crt`
+
+
+### Client key and certificates
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate
+`client.<username>.crt` | Client certificate for `<user>` (e.g., `client.root.crt` for user `root`)
+`client.<username>.key` | Key for the client certificate
+
+## Split CA certificates
+
+{{site.data.alerts.callout_danger}}
+We do not recommend you use split CA certificates unless your organizational security practices mandate you to do so.
+{{site.data.alerts.end}}
+
+If you need to use separate CAs to sign node certificates and client certificates, then we need two CAs and their respective certificates and keys: `ca.crt` and `ca-client.crt`.
+
+### Node key and certificates
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate to sign node certificates
```

Comment:
let's use "to verify" instead of "to sign". The CA does the signing, we're only verifying things.